clang/lib/DPCT/GenMakefile.cpp

```diff
@@ -96,8 +96,8 @@ static void getCompileInfo(
           // Set the target name
           TargetName = Obj;
           IsTargetName = false;
-          Tool = "dpcpp -o"; // Record the tool that generates the target file
-                             // and use dpcpp as the linker tool in the
+          Tool = "$(CC) -o"; // Record the tool that generates the target file
+                             // and use 'icpx' as the linker tool in the
```

Comment:
please test if "icpx" can link sycl object file (or, does it need icpx -fsycl).

Reply:
From experiment, "icpx" cannot link sycl object file without option -fsycl